src/homepage2/HomePage.js

```diff
@@ -0,0 +1,207 @@
+// @flow
+
+import React from "react";
+
+import type {Assets} from "../webutil/assets";
+import Link from "../webutil/Link";
+import {StyleSheet, css} from "aphrodite/no-important";
+
+export default class HomePage extends React.Component<{|+assets: Assets|}> {
```

Comment:
Do we really want to have all this stuff when there's the docs repo as the new homepage?